web/src/js/ducks/settings.js

```diff
@@ -1,64 +1,72 @@
-import { addLogEntry } from './eventLog'
+import {fetchApi} from "../utils";
 
-export const WS_MSG_TYPE = 'settings'
-export const WS_MSG_CMD_UPDATE = 'update'
```

Comment:
As I said, these constants are options but not action type, so I think we should keep them here